crates/astria-sequencer/src/sequence/action.rs

```diff
@@ -27,6 +27,8 @@ use crate::{
 /// Fee charged for a sequence `Action` per byte of `data` included.
 pub(crate) const SEQUENCE_ACTION_FEE_PER_BYTE: Balance = Balance(1);
 
+const MAX_CHAIN_ID_LENGTH: usize = 32;
```

Comment:
Should we make this match the max celestia namespace? See [here](https://github.com/astriaorg/astria/blob/main/crates/astria-celestia-jsonrpc-client/src/rpc_impl/blob.rs#L42C54-L42C54)

Reply:
depends if we want to start using the chain_id here as the namespace directly, currently we're hashing the chain_id and taking the first 10 bytes. I wanted to keep them separate so that chain_id is human readable while namespace might not be, but open to changing it, it's somewhat arbitrary